### PR TITLE
LibrarieJS Updates

### PR DESCRIPTION
**Pull request by ramramps**
This PR 

1. Revert the changes for auto-expand on category.
2. Add-ons are not clickable. Auto expanded by default.
3. Add-Ons icon change 

Note: I did not remove the other changes, but commented out.

screen shot 2018-01-26 at 9 30 15 am

@QilongTang

### CODE REVIEW

**Comment by QilongTang** [comment]
LGTM

File: __tests__/UITests.tsx [hunk 1]
```diff
@@ -335,7 +335,7 @@ describe("LibraryContainer UI", function () {
     }, 500);
   });
 
-  it("coregroup items should auto expand", function () {
+  xit("coregroup items should auto expand", function () {
```
Comment (QilongTang) [inline __tests__/UITests.tsx]: Is it possible to test add-on elements, seems they are not currently in the json so probably not doable right now?

Comment (ramramps) [reply __tests__/UITests.tsx]: yup...now I am trying to add a json to this and see whether a unit test is possible.

**Comment by jnealb** [comment]
@ramramps @QilongTang Sid and I have plans to tackle the LibraryJS testing automation task.

File: __tests__/UITests.tsx (hunk 1 above)
Comment (ramramps) [reply __tests__/UITests.tsx]: @QilongTang Added a test :)

Comment (QilongTang) [reply __tests__/UITests.tsx]: Thank you @ramramps Looks good

**Pull request merged**